compileAndPrepForTests.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Uncomment the following line to delete all artifacts and build from scratch
+# rm -rf node_modules/ .vscode-test/ dist/ server/dist/ server/node_modules/ server/out/
+
+npm install
```

Comment:
Can this just be a script in package.json instead? My impression is that's the convention for Node projects, much like making custom tasks in build.gradle. Same goes for the other .sh file as well. That also keeps things platform-independent.